docs/src/architecture/data-model.md

```diff
@@ -0,0 +1,205 @@
+
+# thin-edge Data Model
+
+The **data model** identifies all data send or received from/to **thin-edge** and its components, to interact with those.
+For all data it defines format and explains behaviour.
+
+## Telemetry Data
+
+**Telemetry Data** consists of **measurements**, **events** and **alarms**. Each is defined by a set of data-elements, each with specific behaviour.
+
+### Measurements
+**measurements** carry values from physical **Sensors**[^1] or a device's **Domain Application**[^1];
+e.g. voltage and current of an electricity meter, or current state of the manufacturing control process
+
+
+A measurement can carry a **single value**, or **muliple values** all taken at a single point in time.
+```javascript
+   {
+     // example for a single-value messurement
+     "temperature":               // 'type_name' of that measurement
+                    25.3,         // 'value' of that measurement
+     "time": "2020-10-15T05:30:47+00:00",  // optional 'timestamp' of that measurement
+   }
+```
+```javascript
+   {
+     // example for a multi-value measurement
+     "current": {                // 'type_name' of that measurement
+        "L1": 9.5,               // the 1st 'value' of that measurement, named as "L1"
+        "L2": 1.3                // the 2nd 'value' of that measurement, named as "L2"
+        // ...even more values can occur
+    },
+     "time": "2020-10-15T05:30:47+00:00",  // optional 'timestamp' of that measurement
+   }
+```
+
+
+|Reference  |Description|
+| --------- | --------- |
+|`type_name`  |a string that identifies the measurement uniquely in context of the device|
+|`value`      |the value that was sampled; can be named (especially in context of a multi-value measurement) or unnamed; must be an integer or floating point number|
+|`timestamp`  |optional time that indicates when values were sampled; when not provided, thin-edge.io uses the current system time as the time of the sample; when provided must be conform to ISO 8601|
+
+  * **behaviour of a measurement:**
+    - thin-edge does not store any historical sampled values for measurements
+    - there is no initialization value for measurements; i.e. a measurement is not visible on thin-edge before the 1st sample was sent to thin-edge
```

Comment:
```suggestion
**behaviour of a measurement**
- thin-edge does not store any historical sampled values for measurements
- there is no initialization value for measurements; i.e. a measurement is not visible on thin-edge before the 1st sample was sent to thin-edge
```

Reply:
The behaviour section header does not need to be a list item itself (only its contents)